packages/aws-cdk/README.md

```diff
@@ -746,6 +746,32 @@ NOTICES
 If you don’t want to see a notice anymore, use "cdk acknowledge <id>". For example, "cdk acknowledge 16603".
 ```
 
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ CDK migrate is currently experimental. We make no guarantees about the outcome or stability of the Command or it's output.
+
+Generates a CDK application from an existing Cloudformation template in any of the CDK supported languages. 
+Required Arguments:
+* `--from-path <my_file_path>` - Takes the relative filse path to the JSON or YAML template of your choice
+* `--stack-name <my_stack_name>` - Used to name both the CDK application and stack. 
```

Comment:
How is it used? Is it used verbatim? Is there a transformation that's applied? Is the CDK App name === the stack name?

Reply:
At this point the CDK App name === the stack name. We want to change this later. I'll make this more explicit